runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -200,50 +201,63 @@ public Dimension render(Graphics2D graphics)
 			}
 
 			final Color color = plugin.getItemColor(highlighted, hidden);
-			itemStringBuilder.append(item.getName());
 
-			if (item.getQuantity() > 1)
+			if (config.highlightTiles())
 			{
-				if (item.getQuantity() >= MAX_QUANTITY)
-				{
-					itemStringBuilder.append(" (Lots!)");
-				}
-				else
+				final Polygon poly = Perspective.getCanvasTilePoly(client, groundPoint);
+				if (poly != null)
 				{
-					itemStringBuilder.append(" (")
-							.append(StackFormatter.quantityToStackSize(item.getQuantity()))
-							.append(")");
+					OverlayUtil.renderPolygon(graphics, poly, color);
 				}
 			}
 
-			if (config.priceDisplayMode() == PriceDisplayMode.BOTH)
+			if ((config.itemHighlightMode() != MENU) || plugin.isHotKeyPressed())
```

Comment:
invert this if so the diff will be more readable and you would not have giant wrapped if. (so if config.itemHighlightMode == MENU || !plugin.isHotkeyPressed()) continue. you should probably also save that at the start of that method and reuse it in both the start check and check here